src/Controllers/AboutController.ts

```diff
@@ -70,6 +63,7 @@ module StreamStats.Controllers {
             this.regionService = region;
             this.selectedAboutTabName = "about";
             this.regionArticle = '<h3>No State or Region Selected</h3>';

```

Comment:
I think we could omit this "No State or Region Selected" unless we change that tab's content based on the state/region selected.

Reply:
That would get displayed if a use selected say the Texas 'i' icon. 
There is another option if a user selects a region that doesnt have a url, say Virgina.

Reply:
Sorry, I must have been looking at something incorrectly. Please ignore!